shoop_tests/core/test_methods.py

```diff
@@ -152,23 +152,6 @@ def test_weight_limits():
     assert any(ve.code == "max_weight" for ve in sm.get_validation_errors(source))
 
 
-# TODO: (TAX) Taxing of shipping methods
-@pytest.mark.xfail # TODO: (TAX) Make this test not fail
```

Comment:
Well not sure what this is testing? Also ShippingMethod without tax_class can't be done at all. With @suutari-ai we though that since this test is poor and not sure what this is actually testing it should be removed.